### ISSUE TITLE
Enhance Label accessibility with ARIA W3C Recommendation and other approaches

### ISSUE THREAD

**Issue by nikiluk** [OpenEthicsAI/OETP#21]
Use best practices in WCAG2.1 to improve the SVG files of the label and their presentation within the OEL. 

This task requires updated label files to be shared on GitHub as they are not yet there, ask @nikiluk 

```
Feature: Accessible labels

Given the label that is displayed in the web interface, loading from label.openethics.ai
When the labels are accessed by the screen readers
Then their meaning should be clear
And the UI should be not only visible but audible
```

**Comment by nikiluk** [comment]
See for example https://css-tricks.com/accessible-svgs/

**Comment by iyanuashiri** [comment]
Hello. I just joined Open Ethics. I was assigned to this task. I think I understand the task. I am to update the oel.html page with accessibility. I have implemented it. I just tried to send a PR, but I dont have write access yet.

**Comment by nikiluk** [comment]
@iyanuashiri please have a look at the repository and how the SVG elements should be updated https://github.com/OpenEthicsAI/OEL
